nailgun/entities.py

```diff
@@ -2443,6 +2444,7 @@ def __init__(self, server_config=None, **kwargs):
             'hosts_available_count': entity_fields.IntegerField(),
             'issued': entity_fields.DateField(),
             'packages': entity_fields.DictField(),
+            'module_streams': entity_fields.DictField(),
```

Comment:
```suggestion
            'module_streams': entity_fields.OneToManyField(ModuleStream)
```
Can you please try this once ?

Reply:
Done!